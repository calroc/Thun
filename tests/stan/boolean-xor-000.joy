true false _\/_
false true _\/_
false false _\/_
true true _\/_
