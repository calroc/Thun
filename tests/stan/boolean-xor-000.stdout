true
true true
true true false
true true false false
